java/core/src/main/java/sleeper/core/properties/instance/CdkDefinedInstanceProperty.java

```diff
@@ -63,6 +63,16 @@ static List<CdkDefinedInstanceProperty> getAllInGroup(PropertyGroup group) {
             .propertyGroup(InstancePropertyGroup.COMMON)
             .build();
 
+    //AWS Config
+    CdkDefinedInstanceProperty ACCOUNT = Index.propertyBuilder("sleeper.account")
+            .description("The AWS account number. This is the AWS account that the instance is be deployed within.")
```

Comment:
There's a typo at "is be deployed", it should be "is deployed".

I'd prefer "in" to "within", it's shorter and easier to read.